src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1066,4 +1071,28 @@ function buildDotNetListItem(item: ListItem): DotNetListItem | null {
         children: children,
         actionSections: item.actionsSections
     } as DotNetListItem;
+}
+
+function checkConnectivity(viewId) {
+    let connectError = new Error('Cannot load ArcGIS Assets!');
+    let message = '<div><h1>Cannot retrieve ArcGIS asset files.</h1><p><a target="_blank" href="https://docs/geoblazor.com/assetFiles"</p></div>';
+    let mapContainer = document.getElementById(`map-container-${viewId}`)!; 
+    try {
+        fetch(esriConfig.assetsPath)
```

Comment:
This seems fine, but are there any issues with calling `fetch` like this if the `assetsPath` is a directory? For example, if the server has list directory contents as false, will this fail? Maybe we should check for a specific file within `assetsPath` since we know it *has* to look like an esri assets folder or nothing will work.

Reply:
Added a path to an actual file